auth/connections.go

```diff
@@ -0,0 +1,50 @@
+package auth
+
+import (
+	"errors"
+	"net"
+	"net/http"
+	"time"
+
+	"golang.org/x/oauth2"
+)
+
+// DefaultTransport is a custom implementation of the
+// DefaultTransport from the net/http stdlib. It limits
+// the IdleConnTimeout to 10 seconds instead of 90 seconds.
+var DefaultTransport http.RoundTripper = &http.Transport{
```

Comment:
Let's throw the file that we copied the DefaultTransport from here!

```suggestion
//
// from "net/http/transport.go"
var DefaultTransport http.RoundTripper = &http.Transport{
```